requirements_all.txt

```diff
@@ -919,6 +919,11 @@ pyhiveapi==0.2.14
 # homeassistant.components.homematic
 pyhomematic==0.1.51
 
+# homeassistant.components.homeworks
+# homeassistant.components.binary_sensor.homeworks
```

Comment:
Run `script/gen_requirements.py` again and commit the changes to update this file.